Xamarin.PropertyEditing/ViewModels/PropertyGroupViewModel.cs

```diff
@@ -55,10 +55,35 @@ public string FilterText
 			}
 		}
 
-		private readonly PropertyViewModel[] properties;
+		public void Add (PropertyViewModel property)
+		{
+			if (property == null)
+				throw new ArgumentNullException (nameof(property));
+
+			this.properties.Add (property);
```

Comment:
So why is it not added to `propertiesView`?

Reply:
`Reload()` handles that.